### PR TITLE
Edge. Added a note sugggested by Chris.

### PR DESCRIPTION
**Pull request by sharath-jk**
This needs to be merged with develop and cherry-picked to release/r10.11.0 branch.

### CODE REVIEW

File: content/edge/configuration-bundle/configuring-microservices.md [hunk 1]
```diff
@@ -19,13 +19,12 @@ If you want to use the microservice hosting feature, ensure that you do not use
...
 	- Username: management/<*username*>
 	- Password: password provided during the installation
-
 2. Switch to the **Administration** application using the application switcher at the right of the top bar **<img class="Default" src="/images/icons/switcher-icon.png" alt="icon" style="display: inline; float: none">**.
-
 3. Click **Edge** > **Microservices** in the navigator.
-
 4. Use the toggle button to enable the microservice hosting feature.
 
+>**Important:** If you have enabled remote-connectivity and also microservice hosting feature, disabling the microservice hosting feature or reconfiguring the network or updating the Edge appliance might result in an alarm in the remote tenant for the Kubernetes network interfaces that are removed or altered.
```
Comment (BeateRixen) [inline content/edge/configuration-bundle/configuring-microservices.md:26]: and also microservice hosting feature > and also the microservice hosting feature

Comment (MWindel) [reply content/edge/configuration-bundle/configuring-microservices.md]: I added your suggestion to my language review

Comment (sharath-jk) [reply content/edge/configuration-bundle/configuring-microservices.md]: Thank you, Marlene.

**Comment by sharath-jk** [comment]
If there are no further changes, could you please approve?

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**